MIGRATION.md

```diff
@@ -1,6 +1,51 @@
 
 # Migration
 
+## Migrating from v2 to v3
+
+o-tracking now uses the Beacon API in all browsers which support it. In v2 this was an opt-in feature, now it is always enabled in browsers which support it. For browsers which do not support Beacon API o-tracking will revert to the XMLHTTPRequest API.
```

Comment:
... there were good reasons not to do this in the past, namely that you can't detect network failures and attempt resending. At the time the trade off between
- beacon will always send, but may not succeed and we won't be able to retry
- XHTTP may not always send, but if the user comes back we get to retry

resulted in a decision to stick with XHTTP (IIRC when we switched to beacon on ft.com we saw a noticeable change in analytics). I'm not sure if that is still the right decision, but has this been discussed with the data platform?

(Incidentally, this is an excellent example of why it'd important to transfer ownership of o-tracking to a team that is more invested in the entire e2e tracking pipeline)

Reply:
Data Platform have seen a more than double increase in submitted events by using the Beacon API versus the XMLHTTPRequest API.

Reply:
wow

Reply:
This is reference to turning on Beacon API and turning off XMLHTTPRequest API:
>We see page interaction events coming from 68% of article page views (not last page) compared to 46% and for the last page of visits we now have 46% compared to 19%.  So more than doubled the delivery on the last page and about a 50% improvement elsewhere.

:D